workshop-butler/public/js/registration-page.js

```diff
@@ -35,6 +37,18 @@ function register_attendee(e) {
             jQuery('#wsb-success').show();
             $form.hide();
             $form.removeClass('.h-busy');
+
+            if( wsb_ga_key != '' ){
+                (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
This part requires a check. Google Analytics may be already on the page. So check 'ga' function for existance first before adding it.